### ISSUE TITLE
Communties -> Category: pasting emoji from exchange buffer should be forbidden

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#11812]
### Description

We dont allow emojis in category names however pasting them CTR+C -> CTR+V works. Please, remove this ability

<img width="1512" alt="Screenshot 2023-08-07 at 13 55 37" src="https://github.com/status-im/status-desktop/assets/82375995/11d5293f-bd47-4f60-91c0-5ef696a7fbaa">

<img width="1512" alt="Screenshot 2023-08-07 at 13 55 42" src="https://github.com/status-im/status-desktop/assets/82375995/a0c8b841-4c6b-4038-8452-108e1a163940">

**Comment by shinnok** [comment]
@anastasiyaig Hi,

Unfortunately I could not reproduce the issue on macOS Ventura with the following emoji copy-paste methods:

1. **Copy pasting native via native emojis**
Invoking `⌃ ⌘ Space` inserts proper unicode emojis from the OS, however, this is caught by our text handler (`alphanumericalExpandedRegExp`) and the  **Save** button is disabled.
<img width="496" alt="SCR-20230821-ltbz" src="https://github.com/status-im/status-desktop/assets/544446/4b70367d-62fd-4d1f-b9d1-988e3715087f">

2. **Copy pasting from within Status app**
Qt/QML doesn't copy proper unicode emoji's into the system clipboard and, resulting in what looks like just plain _white-space_, also when pasting in a different app like **TextEdit**.
<img width="499" alt="SCR-20230821-lucr" src="https://github.com/status-im/status-desktop/assets/544446/b151ee92-eca3-45e1-ac25-20d05fe4172c">

---

I've tested the two copy-paste flows in several other places, like _Add/Edit Channel_, resulting in the same behavior.

### TLDR

Could you please re-test this on your end and get back with how you copy-pasted the _native looking_ emoticon or what extra incantations you did in the edit label, so that the **Save** button stays enabled?

**Comment by anastasiyaig** [comment]
@churik can u please help here? I am on my leave now without laptop :)

**Comment by churik** [comment]
cannot reproduce on latest nightly
